src/main/java/oof/CommandParser.java

```diff
@@ -95,8 +97,14 @@ public static Command parse(String line) throws OofException {
         case "threshold":
             line = line.replaceFirst("threshold", "").trim();
             return new ThresholdCommand(line);
+        case "pause":
+            line = line.replaceFirst("pause", "").trim();
+            return new PauseTrackerCommand(line);
         case "semester":
             return new SemesterCommand();
+        case "viewTracker":
+            line = line.replaceFirst("viewTracker", "").trim();
+            return new ViewTrackerCommand(line);
```

Comment:
Refrain from using camelcase for your commands as it is inconsistent with our past practices

Reply:
This issue has been rectified as suggested.